README.md

```diff
@@ -54,21 +52,15 @@ module.exports = function (environment) {
 };
 ```
 
-### Fastboot
-
-When working with other addons that use `ember-cli-head`, you'll need to create a custom `head.hbs` file that exposes the `<title>` tag properly:
-
-```hbs
-<title>{{model.title}}</title>
-```
-
-This file is added automatically if you use `ember install`. This is for all the folks using ember-cli-head addons like ember-cli-meta-tags.
-
 ### Deprecations
 
 - Since **v5.2.2**: The `{{title}}` helper has been deprecated, use `{{page-title}}` instead, it has the same API. The
   `{{title}}` helper was an AST transform and will be removed in the next major release.
 
+### Upgrading notes for 5.x to 6.x
+
+From 6.x onward, you don't need to have `{{head-layout}}` within your application's `application.hbs` template. If you have `ember-cli-head` as direct dependency on your project, but not used for anything else you can also remove it.
```

Comment:
Maybe mention something like:

> ember-page-title no longer requires the usage of ember-cli-head

At the beginning of the existing prose?

Reply:
I simplified it.